src/core_plugins/metrics/public/components/__tests__/add_delete_buttons.js

```diff
@@ -0,0 +1,67 @@
+// import React from 'react';
+// import { expect } from 'chai';
+// import { shallow } from 'enzyme';
+// import sinon from 'sinon';
+// import AddDeleteButtons from '../add_delete_buttons';
+// import Tooltip from '../tooltip';
+
+// describe('<AddDeleteButtons />', () => {
```

Comment:
(Oh, saw the comment in the backport. I still think this should have been fixed first though. Broken windows and all that.)